__tests__/app.test.js

```diff
@@ -21,3 +28,54 @@ describe("CORE: GET /api/topics", () => {
       });
   });
 });
+
+describe("8 CORE: PATCH /api/articles/:article_id", () => {
+  const patchData = { inc_votes: 1 };
+  it("200: returns article with updated votes, when passed a valid number of an article in request body", () => {
+    return request(app)
+      .patch("/api/articles/7")
+      .send(patchData)
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+
+        articles.forEach((article) => {
+          expect(article).toEqual({
+            article_id: 7,
+            title: "Z",
+            topic: "mitch",
+            author: "icellusedkars",
+            body: "I was hungry.",
+            created_at: "2020-01-07T14:08:00.000Z",
+            votes: 1,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          });
+        });
+      });
+  });
```

Comment:
Need to return the single article updated - at the moment, looks like you're getting an array